docker-compose.dev.yml

```diff
@@ -21,11 +21,13 @@ services:
   spot:
     image: samply/rustyspot:main
     ports:
-      - 8080:8080
+      - 8055:8055
     environment:
-      BEAM_SECRET: "${LOCAL_BEAM_SECRET_SPOT}"
+      RUST_LOG: "info"
+      CORS_ORIGIN: http://localhost:5173
+      BEAM_SECRET: "${LOCAL_BEAM_SECRET_FOCUS}"
       BEAM_PROXY_URL: http://beam-proxy:8081
-      BEAM_APP_ID: "spot.${LOCAL_BEAM_ID}.${BROKER_HOST}"
+      BEAM_APP_ID: "focus.${LOCAL_BEAM_ID}.${BROKER_HOST}"
```

Comment:
Why are you renaming spot to focus?

Reply:
i thought we did this together. was that also a local / prod thing? i usually don't temper with the dockerfiles on my own...

Reply:
i just checked the production configuration, there we still have "spot" as id. So i would change it back. Can you also change back `APP_focus_KEY` to `APP_spot_KEY` in the beam-proxy environment section?